pep-0514.txt

```diff
@@ -118,17 +138,22 @@ It is not possible to detect side-by-side installations of both 64-bit and
 current user. Python 3.5 and later always uses different Tags for 64-bit and
 32-bit versions.
 
-Environments registered under other Company names must use distinct Tags to
-support side-by-side installations. Tools consuming these registrations are
-not required to disambiguate tags other than by preferring the user's setting.
+The following section describe user-visible information that may be registered.
+For Python 3.5 and earlier, none of this information is available, but
+alternative defaults are specified for the ``PythonCore`` key.
+
+Environments registered under other Company names have no backward compatibility
+requirements and must use distinct Tags to support side-by-side installations.
+Tools consuming these registrations are not required to disambiguate tags other
+than by preferring the user's setting.
 
 Company
 -------
 
 The Company part of the key is intended to group related environments and to
 ensure that Tags are namespaced appropriately. The key name should be
 alphanumeric without spaces and likely to be unique. For example, a trademarked
-name, a UUID, or a hostname would be appropriate::
+name (preferred), a UUID, or a hostname would be appropriate::
 
```

Comment:
I'm still strongly -1 on a UUID. The ```virtualenv``` use case implies that this tag is expected to be used by the user, and UUIDs don't work in that context. (Nor do hostnames, frankly).

I'd suggest requiring this to be a short user-friendly abbreviation or name (ideally without spaces, as it's to be used on the command line). Examples could be "conda", "enthought", "jython", "pypy", "ironpython". If you want a longer form, such as your full company name, then the ```DisplayName``` value is what you should use.

I don't see name squatting as a real issue. If "Fred's Distribution" chooses to use "conda" as a company name, it simply won't work if installed alongside Anaconda. People will simply stop using it, because it "breaks their Anaconda installation" and the problem will be self-solving.

It's not like there will be so many distributions that this is going to be a big issue. Heck, the list I gave above is probably the majority of cases we'll see in reality.